src/Model/SearchData.php

```diff
@@ -43,6 +43,18 @@ class SearchData
      */
     private $selectedCreator;
 
+    /**
+     * @var array|null associative array of statuses (key: status int, value: count)
+     */
+    private $statuses;
+
+    /**
+     * @var int|null $selectedStatus
+     */
+    private $selectedStatus;
+
+    private $selectedStatusWidget;
```

Comment:
Viell. ist dies besser als nichts(?):
```suggestion
    /**
     * @var object $selectedStatusWidget
     */    private $selectedStatusWidget;
```